Gemfile.lock

```diff
@@ -359,4 +379,4 @@ RUBY VERSION
    ruby 2.4.4p296
 
 BUNDLED WITH
-   1.17.3
+   2.0.2
```

Comment:
What is our status with production support for `bundler` releases?  Is https://github.com/pulibrary/princeton_ansible/blob/master/roles/pulibrary.ruby/README.md#bundler still accurate @tampakis and @christinach?

Reply:
I don't think this is an issue anymore, not sure why, but the application did deploy fine running bundler 2.0.2
The server currently has ruby 2.4.5 install